dashboard/app/models/aichat_session.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: aichat_sessions
+#
+#  id                   :bigint           not null, primary key
+#  user_id              :integer
+#  level_id             :integer
+#  script_id            :integer
+#  project_id           :integer
```

Comment:
Are we planning to convert `channel_id` to `project_id`?

Reply:
Yes, was imagining using `storage_decrypt_channel_id`? I think by default when saving a project identifier in a Rails model, we should be saving `project_id` rather than `channel_id` -- makes for easier lookups of any project metadata we might need (should be as simple as `aichat_session.project...`), and the word channel confuses me!

Reply:
Sounds good - thanks for the explanation!

Reply:
This would be the right thing to do - we don't want to store channel_ids in any new tables going forward. (we had originally been doing that in datablock storage and infra strongly suggested against it)